helpdesk/templates/helpdesk/ticket_resolves_add.html

```diff
@@ -0,0 +1,37 @@
+{% extends "helpdesk/base.html" %}{% load i18n %}
+
+{% block helpdesk_title %}{% trans "Add Ticket as Dependency" %}{% endblock %}
+
+{% block helpdesk_breadcrumb %}
+<li class="breadcrumb-item">
+    <a href="{% url 'helpdesk:list' %}">{% trans "Tickets" %}</a>
+</li>
+<li class="breadcrumb-item">
+    <a href="{% url 'helpdesk:list' %}{{ depends_on.id }}/">{{ depends_on.queue.slug }}-{{ depends_on.id }}</a>
+</li>
+<li class="breadcrumb-item active">{% trans "Add Ticket as Dependency" %}</li>
+{% endblock %}
+
+{% block helpdesk_body %}{% blocktrans %}
+<h2>Add Ticket as Dependency</h2>
+
+<p>Ticket: {{ depends_on }} </p>
+
+<p>Adding this ticket as dependency will stop you resolving the selected ticket until this ticket has been resolved or closed.</p>{% endblocktrans %}
+
+<form method='post' action='./'>
+
+<fieldset>
+    <dl>{% for field in form %}
+        <dt><label for='id_{{ field.name }}'>{{ field.label }}</label></dt>
+        <dd>{{ field }}</dd>
+        {% if field.errors %}<dd class='error'>{{ field.errors }}</dd>{% endif %}
+        {% if field.help_text %}<dd class='form_help_text'>{{ field.help_text }}</dd>{% endif %}
+    {% endfor %}</dl>
+</fieldset>
+
+<input class="btn btn-primary" type='submit' value='{% trans "Save Ticket Dependency" %}' />
+
+{% csrf_token %}</form>
+
+{% endblock %}
```

Comment:
Can you show also show a screenshot of this new page, please?

Reply:
Shure! It's almost a hardcopy of `ticket_depends_add.html`, For deleting a "parent" dependencies I could recicly `ticket_depends_del.html`, but needed a new view.

![image](https://github.com/django-helpdesk/django-helpdesk/assets/20359147/cf831725-abc1-4dd4-9404-b85bbc1fa035)